ls1intum/Artemis
# [Bugfix] [Orion] Buttons Clickable While Disabled

### Checklist

### Motivation and Context
Resolves https://github.com/ls1intum/Orion/issues/52.

### Description
I was unable to find a way to stop the clicks from being registered using built-in features (and a small local test worked, meaning Angular normally disables buttons just fine) and therefore resorted to add a custom click handler to filter clicks. I agree this is a bit "hacky", any suggestions for a better solution are welcome.

### Steps for Testing

1. Use Orion, preferrably the latest build (the newest features are not released yet, can be found at "actions")
2. Navigate to any Orion button, that is
    1. Import and submit as student (accessible through the student view of any java programmming exercise)
    2. Import, submit and build locally as instructor (accessible through the course management)
    3. Start assessment and download submission (accessible through the assessment dashboard of any java programming exercise) (only available using lastest build)
3. Click the button and then again while the button is shown as loading
4. Only one action should be invoked
5. Review code
